src/main/java/com/google/cloud/tools/appengine/api/devserver/DefaultRunConfiguration.java

```diff
@@ -48,6 +49,7 @@
   private String defaultGcsBucketName;
   private Boolean clearDatastore;
   private File datastorePath;
+  private Map<String, String> environment;
```

Comment:
yea after discussing with @patflynn I realized what you were getting at. I updated devappserver1 to set these env vars onto the running process. However for devappserver2 it needs to be updated to split them up and passed into the command for --env_var. I'll update.